.github/workflows/generate-site.yaml

```diff
@@ -52,13 +68,15 @@ jobs:
         with:
           app-id: ${{ secrets.id }}
           private-key: ${{ secrets.key }}
-          owner: ${{ matrix.site.owner }}
-          repositories: ${{ matrix.site.name }}
+          owner: ${{ env.OWNER }}
+          repositories: ${{ env.REPO }}
```

Comment:
this is the meat of the change: instead of working on a matrix of values, we take in a single value.